concordium-std/src/impls.rs

```diff
@@ -1,30 +1,37 @@
 use crate::{
+    cell::UnsafeCell,
     collections::{BTreeMap, BTreeSet},
-    convert::{self, TryFrom, TryInto},
+    convert::{self, TryInto},
+    fmt,
     hash::Hash,
+    marker::PhantomData,
     mem, num,
-    prims::*,
+    num::NonZeroU32,
+    prims,
     traits::*,
     types::*,
     vec::Vec,
     String,
 };
-use concordium_contracts_common::*;
+pub(crate) use concordium_contracts_common::*;
 use mem::MaybeUninit;
 
+/// Mapped to i32::MIN + 1.
 impl convert::From<()> for Reject {
     #[inline(always)]
     fn from(_: ()) -> Self { unsafe { num::NonZeroI32::new_unchecked(i32::MIN + 1) }.into() }
 }
 
+/// Mapped to i32::MIN + 2.
 impl convert::From<ParseError> for Reject {
     #[inline(always)]
     fn from(_: ParseError) -> Self {
         unsafe { num::NonZeroI32::new_unchecked(i32::MIN + 2) }.into()
     }
 }
 
-/// Full is mapped to i32::MIN+3, Malformed is mapped to i32::MIN+4.
+/// Full is mapped to i32::MIN + 3,
+/// Malformed is mapped to i32::MIN + 4.
```

Comment:
There actually is, because this makes it show up in documentation on docs.rs.
![Screenshot from 2022-05-01 16-46-21](https://user-images.githubusercontent.com/105212/166151115-6001ab4b-3729-48ee-bf2d-29501f9ef5a5.png)